packages/gestalt-design-tokens/tokens/rounding/base.json

```diff
@@ -0,0 +1,14 @@
+{
+  "rounding": {
+    "0": { "value": "0" },
+    "100": { "value": "4px" },
+    "200": { "value": "8px" },
+    "300": { "value": "12px" },
+    "400": { "value": "16px" },
+    "500": { "value": "20px" },
+    "600": { "value": "24px" },
+    "700": { "value": "28px" },
+    "800": { "value": "32px" },
+    "pill": { "value": "999px" }
```

Comment:
we are doing pill but not circle? circle is 50%

Reply:
Added!